From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Prasun Kumar <prasunk@nvidia.com>
Date: Fri, 17 Feb 2023 16:11:37 +0530
Subject: [PATCH 075/241] NVIDIA: SAUCE: hsierrrptinj: Update Ethernet IP
 instances

Updated number of instances for Ethernet driver:
MGBE: 20
EQOS: 5

Bug 3510686

Signed-off-by: prasunk <prasunk@nvidia.com>
Change-Id: Ia14bb260072fe6546536ea1e5c35bd2f9873378b
Reviewed-by: Harshraj Rao <hmrao@nvidia.com>
Reviewed-on: https://git-master.nvidia.com/r/c/linux-stable/+/2907253

diff --git a/drivers/platform/tegra/tegra-hsierrrptinj.c b/drivers/platform/tegra/tegra-hsierrrptinj.c
index 664e17263496..79cf0b6b13b7 100644
--- a/drivers/platform/tegra/tegra-hsierrrptinj.c
+++ b/drivers/platform/tegra/tegra-hsierrrptinj.c
@@ -64,8 +64,8 @@ struct hsm_error_report_frame {
  * @brief IP Instances
  * OTHER - 1
  * GPU   - 1
- * EQOS  - 1
- * MGBE  - 4
+ * EQOS  - 5
+ * MGBE  - 20
  * PCIE  - 11
  * PSC   - 1
  * I2C   = 10
@@ -76,7 +76,7 @@ struct hsm_error_report_frame {
  * SMMU  - 1
  * DLA   - 2
  */
-static unsigned int ip_instances[NUM_IPS] = {1, 1, 1, 4, 11, 1, 10, 2, 2, 1, 1, 1, 2};
+static unsigned int ip_instances[NUM_IPS] = {1, 1, 5, 20, 11, 1, 10, 2, 2, 1, 1, 1, 2};
 
 /* This directory entry will point to `/sys/kernel/debug/tegra_hsierrrptinj`. */
 static struct dentry *hsierrrptinj_debugfs_root;
-- 
2.43.0

